frontend/src/css/modules/variables.scss

```diff
@@ -30,6 +30,7 @@ $dark-black: #4d4d4d;
 
 /*orange shades*/
 
+$yellow: #ffaf4b;
 $highlight: #ffaf4b;
```

Comment:
@Ram81 both `$highlight: #ffaf4b;` and `$yellow: #ffaf4b;` have the same value. Can you please use one.